src/keyboard.c

```diff
@@ -243,8 +243,9 @@ handle_keybind_repeat(void *data)
 	};
 
 	handle_compositor_keybindings(keyboard, &event);
+	int next_repeat_ms = 1000 / keyboard->keybind_repeat_rate;
```

Comment:
Is it possible that `keyboard->keybind_repeat_rate` is set to `0`? If yes this would be a division by zero.

Reply:
It shouldn't be zero here because we checked it in `start_keybind_repeat()`. Wouldn't hurt to add another `assert()` at the start of this function though.

Reply:
Assert added.